src/components/atomic/dynamic-form/VChecklist.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+    <div>
+        <v-form-label v-bind="labelProps">
+            <div class="mt-1 flex rounded-md shadow-sm">
+                <div v-for="checklist in checklists">
+                    <input
```

Comment:
This ideally wants to load as a list of VCheckList items instead, as otherwise you've got 2 locations to update rather than one :)